templates/CRM/Certificate/Page/CertificateDownload.tpl

```diff
@@ -0,0 +1,39 @@
+{crmScript ext=uk.co.compucorp.certificate file=js/vendor/html2canvas.min.js}
```

Comment:
I see in the PR description you show an example on this, but I wonder if it works fine with HTML that contain images, both local and remote images, have you tested that ?

Reply:
yes works with images. updated PR description.